website/blog/2022-09-28-analyst-to-ae.md

```diff
@@ -0,0 +1,178 @@
+
+### Model format needed for the BI tool
+
+| Date       | Group   | Daily Active Users |
+|------------|---------|--------------------|
+| 2022-08-01 | Group A | 34                 |
+| 2022-08-01 | Group B | 60                 |
+| 2022-08-01 | Group C | 61                 |
+| 2022-08-02 | Group A | 77                 |
+| 2022-08-02 | Group B | 86                 |
+| 2022-08-02 | Group C | 37                 |
+| 2022-08-03 | Group A | 71                 |
+| 2022-08-03 | Group B | 9                  |
+| 2022-08-03 | Group C | 6                  |
+
+## Loop 2: Reacting to unexpected results
+
+Analysts are the first (and sometimes only) line of defense to catch data quality issues. Once a dataset is aggregated down to a single number to answer a business question, it’s often impossible to know if there’s an incorrect filter or wrong set of logic.
+
+When a model is built for the first time, all parties often “don’t know what they don’t know”. As the Analyst digs into the curated dataset, they will find updates that need to be made to the model. Hence, our second point:
+
+<h3 align="center">Analysts question everything.</h3>
+
+`Group A` for `Active Users` may be focused on `Participant` roles. The stakeholder had explained to the Analyst that they want to exclude any `Admins` that will have a different experience in the product.
+
+| User_Id | Location   | Role        | Level | Zone |
+|---------|------------|-------------|-------|------|
+| 123     | California | Editor      | AAA   | 1    |
+| 427     | Utah       | Participant | ABA   | 1    |
+| 864     | Georgia    | Admin       | CCC   | 3    |
+| 975     | New York   | Participant | CAB   | 4    |
+| 211     | Florida    | Participant | CBB   | 3    |
+| 936     | Montana    | Admin       | ABA   | 2    |
+| 787     | Nevada     | Participant | ACC   | 2    |
+
+A Data Engineer working off of a “build list” will add a filter for `WHERE Role = 'Participant'`. During the data validation step, the Analyst would discover that there is actually a third `Role` of `Editor` that no-one was aware of. This would create a loop where the Data Engineer would have to edit the model to replace it with `WHERE Role != 'Admin'`.
+
+The Analyst caught the issue because they have the appropriate context to validate the dataset. Analysts work directly with the stakeholder, so they’ll also be more familiar with company-specific and department-specific acronyms.
+
+**And as we all know, data validation is an art, not a science.** Analysts will employ everything from a “sniff test” (view a random sample of rows) to individual examples (one-to-one exact matching to another system). An Analyst will have to use their experience to know when the dataset is “good enough” for the stakeholder and their question, since 100% accuracy might not be the goal. And if we're being honest, sometimes being directionally correct is all that’s needed to make a business decision.
```

Comment:
```suggestion
**And as we all know, data validation is an art, not a science.** Analysts will employ everything from a “sniff test” (view a random sample of rows) to individual examples (one-to-one exact matching to another system). An Analyst will have to use their experience to know when the dataset is “good enough” for the stakeholder and their question since 100% accuracy might not be the goal. And if we're being honest, sometimes being directionally correct is all that’s needed to make a business decision.
```